spec/integration/timeout_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+require_relative 'fake_bits_service'
+require "webrick"
+require 'net/http'
+require 'spec_helper'
+
+describe BitsService::Client, :integration_test do
+    let(:fake_server){}
+    let(:fake_endpoint){'localhost'}
+    let(:fake_endpoint_port){9292}
+
+    let(:resource_type) { [:buildpacks, :droplets, :packages].sample }
+    let(:key) { SecureRandom.uuid }
+    let(:vcap_request_id) { '4711' }
+
+    let(:options) do
```

Comment:
Consider inlining those options. They're only used in one place. By inlining them, the reader is not forced to jump back and forth between definition and usage. To keep readability high, the line
```ruby
subject(:client) { BitsService::Client.new(bits_service_options: options, resource_type: resource_type, vcap_request_id: vcap_request_id, request_timeout_in_seconds_fast: 1) }
```
Could broken up into multiple lines.

Reply:
I agree and this is changed in 
ca5f427